src/comps/Nav.jsx

```diff
@@ -15,7 +17,7 @@ export default function Nav() {
         <button className={navCss.btn}>CART</button>
       </Link>
       <Link to='/'>
-      <button className={navCss.btn}>LOG OUT</button>
+      <button className={navCss.btn} onClick={()=>valContext.userDisconnect()}>LOG OUT</button>
       </Link>
```

Comment:
You don't need to create an arrow function that calls some function without any parameters.

You can simply write `onClick={valContext.userDisconnect`